doc/03-omnifest/02-external.md

```diff
@@ -11,13 +11,22 @@ JSON again in a specific format. `otk` will look for these directives in
 `/usr/libexec/otk`. Externals are explicitly mapped to executables in the
 registry that is defined in `src/otk/external.py`.
 
-An external gets called with the following JSON:
+When we have an [omnifest](./index.md) that looks like this:
```

Comment:
Bit of a rephrase to avoid "you", "we", etc could be:
```
Given the following omnifest:
...

The external called `name` gets called with the following JSON as input:
...
```